config.py

```diff
@@ -130,6 +131,7 @@ class DevelopmentConfig(Config):
     DEBUG = True
     TEMPLATES_AUTO_RELOAD = True
     SECRET_KEY = os.getenv('SECRET_KEY', 'ONS_DUMMY_KEY')
```

Comment:
The default in party for SECRET_KEY is different (aardvark) and as we are now using it, it will fall over. This is kind of irrelevant as we inject SECRET_KEY into all the important environments, but it could trip up someone running it locally so worth changing

Reply:
Should we delete the default and issues errors for unset environment variables, like you did in Secure Message?

Reply:
Delete default for SECRET_KEY?